cfgov/root/utilities/pages/akamai-sureroute-test-object.htm

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
```

Comment:
Note: this new file is a duplicate of the one at cfgov/root/akamai/sureroute-test-object.html. The production website currently serves this same file at two URLs:

- https://www.consumerfinance.gov/akamai/sureroute-test-object.html
- https://www.consumerfinance.gov/utilities/pages/akamai-sureroute-test-object.htm